README-PT.md

```diff
@@ -0,0 +1,216 @@
+[![Build Status](https://travis-ci.org/malukenho/MigrateDB.png?branch=master)](https://travis-ci.org/malukenho/MigrateDB) [![Latest Stable Version](https://poser.pugx.org/malukenho/migratedb/v/stable.png)](https://packagist.org/packages/malukenho/migratedb) [![Total Downloads](https://poser.pugx.org/malukenho/migratedb/downloads.png)](https://packagist.org/packages/malukenho/migratedb) [![Latest Unstable Version](https://poser.pugx.org/malukenho/migratedb/v/unstable.png)](https://packagist.org/packages/malukenho/migratedb) [![License](https://poser.pugx.org/malukenho/migratedb/license.png)](https://packagist.org/packages/malukenho/migratedb)
+# MigrateDB
+
+
+
+
+**MigrateDB** é uma ferramenta simples para migrar dados entre bancos de dados.
+
+#### 1ª Etapa:
+
+Instalando o  **MigrateDB** fica muito simples usar o composer :3
```

Comment:
You have much spaces after `o` and your phrase have no much sense. I think you mean:

>  É muito simples instalar o **MigrateDB** utilizando o composer :3

or anything else.